cpp/tests/structure/renumbering_test.cpp

```diff
@@ -0,0 +1,176 @@
+
+    if (PERF) {
+      handle.get_stream_view().synchronize();  // for consistent performance measurement
+      double elapsed_time{0.0};
+      hr_clock.stop(&elapsed_time);
+      std::cout << "renumbering took " << elapsed_time * 1e-6 << " s.\n";
+    }
+
+    if (renumbering_usecase.check_correctness) {
+      cugraph::unrenumber_local_int_vertices(handle,
+                                             src_v.data(),
+                                             src_v.size(),
+                                             renumber_map_labels_v.data(),
+                                             0,
+                                             static_cast<vertex_t>(renumber_map_labels_v.size()));
+      cugraph::unrenumber_local_int_vertices(handle,
+                                             dst_v.data(),
+                                             dst_v.size(),
+                                             renumber_map_labels_v.data(),
+                                             0,
+                                             static_cast<vertex_t>(renumber_map_labels_v.size()));
+
+      h_final_src_v.resize(src_v.size());
+      h_final_dst_v.resize(dst_v.size());
+
+      raft::update_host(h_final_src_v.data(), src_v.data(), src_v.size(), handle.get_stream());
+      raft::update_host(h_final_dst_v.data(), dst_v.data(), dst_v.size(), handle.get_stream());
+
+      EXPECT_EQ(h_original_src_v, h_original_src_v);
+      EXPECT_EQ(h_original_dst_v, h_original_dst_v);
+    }
+  }
+};
+
+using Tests_Renumbering_File = Tests_Renumbering<cugraph::test::File_Usecase>;
+using Tests_Renumbering_Rmat = Tests_Renumbering<cugraph::test::Rmat_Usecase>;
+
+// FIXME: add tests for type combinations
+TEST_P(Tests_Renumbering_File, CheckInt32Int32FloatFloat)
+{
+  auto param = GetParam();
+  run_current_test<int32_t, int32_t, float>(std::get<0>(param), std::get<1>(param));
+}
+
+// FIXME: add tests for type combinations
+TEST_P(Tests_Renumbering_Rmat, CheckInt32Int32FloatFloat)
+{
+  auto param = GetParam();
+  run_current_test<int32_t, int32_t, float>(std::get<0>(param), std::get<1>(param));
+}
+
+INSTANTIATE_TEST_SUITE_P(
+  file_test,
+  Tests_Renumbering_File,
+  ::testing::Combine(
+    // enable correctness checks
+    ::testing::Values(Renumbering_Usecase{}),
+    ::testing::Values(cugraph::test::File_Usecase("test/datasets/karate.mtx"),
+                      cugraph::test::File_Usecase("test/datasets/web-Google.mtx"),
+                      cugraph::test::File_Usecase("test/datasets/ljournal-2008.mtx"),
+                      cugraph::test::File_Usecase("test/datasets/webbase-1M.mtx"))));
+
+INSTANTIATE_TEST_SUITE_P(
+  rmat_small_tests,
+  Tests_Renumbering_Rmat,
+  ::testing::Combine(
+    // enable correctness checks
+    ::testing::Values(Renumbering_Usecase{}),
+    ::testing::Values(cugraph::test::Rmat_Usecase(10, 16, 0.57, 0.19, 0.19, 0, false, false))));
+
+INSTANTIATE_TEST_SUITE_P(
+  rmat_large_tests,
```

Comment:
See https://github.com/rapidsai/cugraph/blob/branch-21.10/cpp/tests/prims/mg_count_if_v.cu#L225
We renamed rmat_large_tests to rmat_benchmark_test to use this in C++ benchmarking.

Reply:
Addressed in latest push.